core/src/main/java/org/arquillian/smart/testing/GitRunnerProperties.java

```diff
@@ -0,0 +1,17 @@
+package org.arquillian.smart.testing;
+
+public class GitRunnerProperties {
+
+    public static final String PREVIOUS_COMMIT = "git.previous.commit";
```

Comment:
Can we follow the convention like in `PREVIOUS_COMMIT` here (`git.last.commits`)?